ansible/setup_fedora.yml

```diff
@@ -1,117 +1,191 @@
+        - gobby05
+        - google-roboto-slab-fonts
+        - gstreamer-ffmpeg
+        - gstreamer1-plugins-bad-free-extras
+        - gstreamer1-plugins-bad-nonfree
+        - gstreamer1-plugins-ugly
+        - hexchat
+        - htop
+        - i3
+        - i3lock
+        - inkscape
+        - jpegoptim
+        - jq
+        - libvirt
+        - libvirt-devel
+        - meld
+        - mpv
+        - nautilus-dropbox
+        - network-manager-applet
+        - npm
+        - ntp
+        - overpass-fonts
+        - overpass-mono-fonts
+        - pandoc
+        - pass
+        - pavucontrol
+        - picard
+        - powerline
+        - pulseaudio-utils
+        - python2-flake8
+        - python3-flake8
+        - ripright
+        - spotify-client
+        - strace
+        - sublime-text
+        - task
+        - telegram-desktop
+        - thunderbird
+        - thunderbird-enigmail
+        - tmux
+        - tmux-powerline
+        - typetype-molot-fonts
+        - '@vagrant'
+        - vagrant-doc
+        - vagrant-sshfs
+        - vim
+        - '@virtualization'
+        - which
+    - name: start Docker
+      service:
+        name: docker
+        state: started
+    - name: start and enable NTP
+      service:
+        name: ntpd
+        state: started
+        enabled: yes
+    - name: add new user - jflory
+      user:
+        name: jflory
+        comment: Justin W. Flory
+        groups: libvirt,wheel
+        append: yes
+    - name: configure dotfiles
+      git: repo=https://github.com/jwflory/conf.git dest={{ git_conf_dir }}
+    - name: populate config directories
+      file: path={{ user_home_dir }}/{{ item }} state=directory
+      with_items:
+        - .bash_completion.d
+        - .config/bugwarrior
+        - .config/dunst
+        - .config/i3
+        - .config/powerline
+        - .npm-global
+        - .ssh
+        - .vim/autoload
+        - .vim/bundle
+        - wkspc
+    - name: configure i3wm
+      file: force=yes src={{ git_conf_dir }}/i3wm/{{ item }} dest={{ user_home_dir }}/.config/i3/{{ item }} state=link
+      with_items:
+        - config
+        - fuzzy_lock.sh
+        - i3exit.sh
+        - i3status.conf
+    - name: configure bash
+      file: force=yes src={{ git_conf_dir }}/bash/{{ item }} dest={{ user_home_dir }}/.{{ item }} state=link
+      with_items:
+        - bashrc
+        - bash_profile
+    - name: configure bash auto-completion scripts
+      file: force=yes src={{ git_conf_dir }}/taskwarrior/{{ item }} dest={{ user_home_dir }}/.bash_completion.d/{{ item }} state=link
+      with_items:
+        - task.sh
+    - name: configure bugwarrior
+      file: force=yes src={{ git_conf_dir }}/bugwarrior/bugwarriorrc dest={{ user_home_dir }}/.config/bugwarrior/bugwarriorrc state=link
+    - name: configure dunst
+      file: force=yes src={{ git_conf_dir }}/dunst/dunstrc dest={{ user_home_dir }}/.config/dunst/dunstrc state=link
+    - name: configue git
```

Comment:
s/configue/configure

Reply:
@Tjzabel I can always count on you to catch my typos. :smile: Thanks!